auth/src/main/java/com/firebase/ui/auth/KickoffActivity.java

```diff
@@ -25,7 +25,7 @@
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 public class KickoffActivity extends InvisibleActivityBase {
-    private SignInKickstarter mKickstarter;
+    public static SignInKickstarter mKickstarter;
```

Comment:
Keeping a reference to a static object in Android is a really bad idea, but I see what you meant when you said that state management is hard when we move stuff from KickoffActivity to AuthMethodPicker - I'll let this be for now, and we can change in the future if we re-architect this flow.